Core/RestaurantReservation.Application/Services/LoginService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using RestaurantReservation.Application.Contracts.IService;
+using RestaurantReservation.Application.Contracts.IServices;
+using RestaurantReservation.Application.DTOs;
+using RestaurantReservation.Domain.Entities;
+
+namespace RestaurantReservation.Application.Services
+{
+    public class LoginService : ILoginService
+    {
+        private readonly string _pepper;
+        private readonly int _iteration = 3;
+        private readonly IAuthenticationService _authenticationService;
+
+        private readonly List<User> _userRepository = new List<User>()
```

Comment:
I'd rather implementing a repository that using the in memory collection rather than using the collection inside the login service, that would make it hard to test / extend it.